eng/packages/http-client-csharp/generator/Azure.Generator/src/Primitives/NewAzureProjectScaffolding.cs

```diff
@@ -81,22 +87,23 @@ protected override string GetSourceProjectFileContent()
             "VoidValue.cs"
         ];
 
-        private static void TraverseInput(out bool hasOperation, out bool hasLongRunningOperation)
+        private static void TraverseInput(InputClient rootClient, out bool hasOperation, out bool hasLongRunningOperation)
```

Comment:
Don't we need to use `ref` rather than `out`? Otherwise, we are wiping out the value whenever we enter the recursive call.